fs/ops.go

```diff
@@ -145,6 +215,7 @@ func WithSymlink(path, target string) PathOp {
 // the other functions in this package.
 func WithHardlink(path, target string) PathOp {
 	return func(root Path) error {
+		// TODO: handle path.(manifestDirectory)
```

Comment:
Still TODO ? (same below)

Reply:
replaced these two with an error "not implemented", I'm not sure how they will work with manifests